server/apiRoutes.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+
+const router = express.Router()
+
+const db = require('./db')
```

Comment:
Let's move this line up so all the `require`s are together. Preferably with a blank line after the external `require`s.